packages/office-ui-fabric-react/src/demo/pages/IconPage/examples/IconExample.scss

```diff
@@ -0,0 +1,56 @@
+.ms-IconExample {
+  font-size: 50px;
+  margin-left: 25px;
+  margin-right: 25px;
+}
+
+.ms-IconColorExample-deepSkyBlue {
+  color: deepskyblue;
+}
+
+.ms-IconColorExample-greenYellow {
+  color: #adff2f;
+}
+
+.ms-IconColorExample-salmon {
+  color: salmon;
+}
+
+.ms-IconImageSheetExample-one {
+  width: 48px;
+  height: 44px;
+  margin-left: 27px;
+}
+
+.ms-IconImageSheetExample-one-image {
+  display: inline-block;
+  position: relative;
+  left: -6px;
+  top: -4px;
+}
+
+.ms-IconImageSheetExample-check {
+  width: 35px;
+  height: 43px;
+  margin-left: 55px;
+}
+
+.ms-IconImageSheetExample-check-image {
+  display: inline-block;
+  position: relative;
+  left: -60px;
```

Comment:
This is my only remaining piece of feedback. The current approach works, but it requires writing a CSS class for every icon. It seems simpler to add props to Image that would control the position of the image within the frame (e.g. `imageOriginX` and `imageOriginY`) and then no CSS would be required. It would also be useful functionality for the Image component in general.

This functionality could be implemented later, so I'll go ahead and approve this PR if you want to get it merged with the current method. 😄